Formula/asdf.rb

```diff
@@ -4,30 +4,24 @@ class Asdf < Formula
   url "https://github.com/asdf-vm/asdf/archive/v0.8.0.tar.gz"
   sha256 "9b667ca135c194f38d823c62cc0dc3dbe00d7a9f60caa0c06ecb3047944eadfa"
   license "MIT"
-  revision 1
+  revision 2
   head "https://github.com/asdf-vm/asdf.git"
 
   bottle :unneeded
-
-  depends_on "autoconf"
-  depends_on "automake"
   depends_on "coreutils"
-  depends_on "libtool"
-  depends_on "libyaml"
-  depends_on "openssl@1.1"
-  depends_on "readline"
-  depends_on "unixodbc"
-
-  conflicts_with "homeshick",
-    because: "asdf and homeshick both install files in lib/commands"
+  depends_on "git"
```

Comment:
Does it not work with the one that comes with macOS?

Reply:
Yeah it does, I initially removed all the dependencies, but since it mentions `coreutils` and `git` in the docs, we might as well bring in both, also minimize risking different tools across platforms.